src/views/Header.js

```diff
@@ -227,8 +272,8 @@ class Header extends React.Component<DefaultProps, HeaderProps, *> {
     return (
       <Animated.View {...rest} style={[styles.container, style]}>
         <View style={styles.appBar}>
-          {leftComponents}
           {titleComponents}
+          {leftComponents}
```

Comment:
This is a temporary solution probably until we figure out how to change the layouts here. But since it changes a few things, I'd say let's ship it and I can keep working on back button layout in the meantime.